angular-client/src/components/charging-switch/charging-switch.component.html

```diff
@@ -0,0 +1,5 @@
+<info-background icon="electrical_services" title="Charging">
+  <div class="switch">
+    <switch />
```

Comment:
spacing needs to be better
![image](https://github.com/Northeastern-Electric-Racing/Argos/assets/29521172/def9538e-36c7-42b8-bd97-7eaf27da7bd4)